### PR TITLE
Test1 pull request template

### PR DESCRIPTION
**Pull request by aasthakasera**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md]: add these lines as comment.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md]: this

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Add Screenshots
+
+Please provide screenshots to get the mentor know easily about the changes you have made and how well is it working now?
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md]: this

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Add Screenshots
+
+Please provide screenshots to get the mentor know easily about the changes you have made and how well is it working now? 
+
+## Performance Impact
+
+Please describe any relevant performance impact of this change. This can be positive or negative impact. How did you characterize/test the performance impact?
```
Comment (DreamyPhobic) [inline .github/pull_request_template.md]: and this

**Review by DreamyPhobic — CHANGES_REQUESTED** [review]

**Comment by aasthakasera** [comment]
Kindly check it.

**Pull request merged**